maven/maven.bzl

```diff
@@ -159,11 +219,79 @@ def _should_include_dependency(dep):
         not dep.optional
     )
 
+def _cut(str, *to_phrases):
+    start_idx = 0
+    for phrase in to_phrases:
+        idx = str.find(phrase, start_idx)
+        if idx == -1:
+            return ("", False)
+        start_idx = idx + len(phrase)
+
+    return (str[start_idx:].strip(), True)
+
+def _extract_android_package(ctx, manifest):
+    m = ctx.read(manifest)
+    pkg = ""
+    for tag in m.split("<"):
+        if tag.startswith("manifest"):
+            # matching package\s*=\s*"|'pkg"|'
+            quot_pkg_quot_trailing, ok = _cut(tag, "package", "=")
```

Comment:
this is the most go-like thing I've seen in starlark in a while. :)

Reply:
Got tired of guessing what the error value was. If you get deconstructed assignment and anonymous pairs, you can have go.